nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1266,7 +1291,7 @@ private Map<String, String> convertAttributesFromHeaders(final Response response
             if (!values.isEmpty()) {
                 // create a comma separated string from the values, this is stored in the map
                 final String value = StringUtils.join(values, MULTIPLE_HEADER_DELIMITER);
-                attributes.put(key, value);
+                attributes.put(trimToEmpty(prefix) + key, value);
```

Comment:
Recommend declaring `final trimmedPrefix = trimToEmpty(prefix)` to avoid calling this method on every attribute addition.